tools/disasm/functions.txt

```diff
@@ -15376,11 +15376,11 @@
     0x80BB99F0:("ObjChan_Destroy",),
     0x80BB9A1C:("func_80BB9A1C",),
     0x80BB9B40:("ObjChan_CalculatePotPosition",),
-    0x80BB9C08:("ObjChan_InitMain",),
-    0x80BB9F24:("ObjChan_MainAction",),
+    0x80BB9C08:("ObjChan_InitChandelier",),
+    0x80BB9F24:("ObjChan_ChandelierAction",),
     0x80BBA2FC:("ObjChan_InitPot",),
     0x80BBA314:("ObjChan_PotAction",),
```

Comment:
Hmm, seems like these shouldn't be called `Action` anyway.

Reply:
Personally, I don't really like renaming things in my cleanup PRs, feels like scope creep. Sounds better suited for a more thorough documentation pass.